### PR TITLE
Add new icons for adverse_event and allergies_intolerances

### PR DESCRIPTION
**Pull request by ssayer**
Fixes frontend so new data criteria have a nice shiny icon.

Related to https://github.com/projectcypress/health-data-standards/pull/511

### CODE REVIEW

**Review by holmesie — APPROVED** [review]
Looks good

**Comment by ssayer** [comment]
@holmesie Please re-review now that this PR points at updated HDS and CQL code

File: app/assets/javascripts/models/data_criteria.js.coffee [hunk 1]
```diff
@@ -80,6 +80,8 @@ class Thorax.Models.PatientDataCriteria extends Thorax.Model
     icons =
       characteristic:            'fa-user'
       communications:            'fa-files-o'
+      allergies_intolerances:    'fa-exclamation-triangle'
+      adverse_events:            'fa-exclamation'
```
Comment (c-monkey) [inline app/assets/javascripts/models/data_criteria.js.coffee:84]: Allergies/Intolerances tested and looks great.

We should get a measure to test adverse events.

**Comment by holmesie** [comment]
@ssayer Re-tested, looks good

**Review by c-monkey — APPROVED** [review]
The SHA values in the Gemfile.lock are correct.

The icon looks great.

This is related to https://jira.mitre.org/browse/BONNIE-693

Logic in original bug works if:
- delete and re-upload measure
- edit the "DENEXCEPPass MedAllergyEndIP" patient:
  * remove the "MEDICATION ALLERGY Allergy: Statin Allergen" from patient history
  * add the "ALLERGY INTOLERANCE Allergy/Intolerance: Statin Allergen" to the patient history

Tested and verified:
https://jira.mitre.org/browse/BONNIE-744 regressions.
and
https://jira.mitre.org/browse/BONNIE-785 feature test

**Pull request merged**